recipe/meta.yaml

```diff
@@ -43,30 +49,27 @@ test:
     {% for executable in bzip2_executables %}
     - test -f ${PREFIX}/bin/{{ executable }}  # [unix]
```

Comment:
```suggestion
    - test -f ${PREFIX}/bin/{{ executable }}
```

Reply:
We test on win64 only two executables in `run_test.bat`:
if not exist %LIBRARY_BIN%\bzip2.exe exit 1
if not exist %LIBRARY_BIN%\bzip2recover.exe exit 1

No other exe files are created. Ian has already asked about this issue https://github.com/AnacondaRecipes/bzip2-feedstock/pull/3#discussion_r1500405484. I'm still not sure if we need them